1-JavaScript/Week2/homeworkSolition/ex2-sol-dogYears.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+
+function calculateDogAge(years) {
+  let dogYears = years * 7;
+  let telldogYears = `your dog is ${dogYears} years old in dog years!`;
+  return telldogYears;
+}
```

Comment:
Great work! Also here you can directly return the string:

```suggestion
function calculateDogAge(years) {
  let dogYears = years * 7;
  return `Your dog is ${dogYears} years old in dog years!`;
}
```